projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -39,6 +39,13 @@ public function dequeue_scripts() {
 		wp_dequeue_script( 'a8c_wpcom_masterbar_overrides' ); // Initially loaded in modules/masterbar/masterbar/class-masterbar.php.
 	}
 
+	/**
+	 * Determines whether the current locale is right-to-left (RTL).
+	 */
+	public function is_rtl() {
+		return 'rtl' === get_user_option( 'jetpack_text_direction' );
+	}
+
```

Comment:
WoW, I see it clear now.. That was my fault, sorry. I used `is_rtl` but ignored it was scoped here https://github.com/Automattic/jetpack/blob/9a65c4bb54eb4f2ffdf6b970c05242f2e4ee4a18/projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php#L255-L257

Could we possibly, reuse that function instead of redeclaring it?

Reply:
That'd be great, but we don't have access to the `Masterbar` instance at this point, so in order to reuse the function we would need:
- Instantiate it again. Not sure about the side effects of this, the constructor registers a few filters/actions that may run twice with unpredictable results.
- Pass the previous instance as param when loading the admin menu class. We'd need to slighter refactor all the admin menu classes so the `get_instance` method receives now a `Masterbar` instance. This options seems a bit tangential for the scope of this PR.
- Refactor the `Masterbar` class so `is_rtl` is a static method we can call it without instantiate it. Not trivial since it depends on data set in the constructor, so not sure it should be done on this PR.

Do you see any other way of reusing this function easily?

Reply:
Thanks for explaining it thoroughly. In that case, given that `is_rtl()` footprint is minor, let's move on with the current implementation!

Reply:
I have simplified a little bit the implementation of the `is_rtl` function in fb884e3, so at least we have less duplicate code now.